macosx/Compatibility.m

```diff
@@ -0,0 +1,20 @@
+//
+//  Compatibility.m
+//  Transmission
+//
+//  Created by Antoine Coeur on 31/03/2022.
+//  Copyright © 2022 The Transmission Project. All rights reserved.
+//
+
+#import "Compatibility.h"
+
+#pragma GCC diagnostic ignored "-Wincomplete-implementation"
+#pragma GCC diagnostic ignored "-Wobjc-property-implementation"
+#pragma GCC diagnostic ignored "-Wprotocol"
+
+#ifndef __MAC_11_0
+
+@implementation NSImageSymbolConfiguration
+@end
```

Comment:
I believe It would crash because of missing selector.
But that scenario is only for someone building the app themselves: the nightly builds and the official builds wouldn't suffer from being built on a two years old Xcode.
Someone building the app themselves could just rebuild it again after upgrading their OS to Big Sur or newer.

Reply:
I've deleted the file and moved the `#ifdef __MAC_11_0` to NSImageAdditions.m instead: like that, it will support your scenario.